tests/tpch/test_dask.py

```diff
@@ -306,3 +306,49 @@ def test_query_7(client, dataset_path, fs):
         by=["supp_nation", "cust_nation", "l_year"],
         ascending=True,
     ).compute()
+
+
+@pytest.mark.shuffle_p2p
+def test_query_13(client, dataset_path, fs):
+    """
+    select
+        c_count, count(*) as custdist
+    from (
+        select
+            c_custkey,
+            count(o_orderkey)
+        from
+            customer left outer join orders on
+            c_custkey = o_custkey
+            and o_comment not like '%special%requests%'
+        group by
+            c_custkey
+        )as c_orders (c_custkey, c_count)
+    group by
+        c_count
+    order by
+        custdist desc,
+        c_count desc
+    """
+    customer = dd.read_parquet(dataset_path + "customer", filesystem=fs)
+    orders = dd.read_parquet(dataset_path + "orders", filesystem=fs)
+
+    subquery = customer.merge(orders, left_on="c_custkey", right_on="o_custkey")
```

Comment:
This should be a left (outer) join
```suggestion
    subquery = customer.merge(orders, left_on="c_custkey", right_on="o_custkey", how="left")
```